satellite/console/consoleweb/introspect_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+package consoleweb_test
```

Comment:
Empty line above package, otherwise it'll be interpreted as a package documentation.